app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -124,10 +124,9 @@ def bypass_send_link_steps
       # for the 50/50 state
       flow_session['Idv::Steps::UploadStep'] = true
 
-      response = form_response(destination: :document_capture)
-      analytics.idv_doc_auth_upload_submitted(
-        **analytics_arguments.merge(response.to_h),
-      )
+      analytics_args = analytics_arguments.merge(form_response(destination: :document_capture).to_h)
+      analytics_args[:flow_path] = flow_session[:flow_path]
```

Comment:
question: if `flow_session[:flow_path]` couldn't be anything other than `'standard'` as assigned in line 121 above, could you assign `'standard'` here directly? (right now what's being assigned here is a bit obfuscated, but you may genuinely not know)